tests/beam_evolution.1Rank.sh

```diff
@@ -40,7 +40,7 @@ mpiexec -n 1 $HIPACE_EXECUTABLE $HIPACE_EXAMPLE_DIR/inputs_normalized \
         beam.density = 1.e-8 \
         beam.radius = 1. \
         beam.ppc = 4 4 1 \
-        hipace.external_ExmBy_slope = .5 \
+        hipace.external_E_slope = .5 0. 0. \
```

Comment:
Ah, I misinterpreted the behavior of the previous implementation
```suggestion
        hipace.external_E_slope = .5 .5 0. \
```